core/src/main/java/com/datastax/oss/driver/internal/core/channel/ChannelHandlerRequest.java

```diff
@@ -71,7 +71,9 @@ private void writeListener(Future<? super Void> writeFuture) {
       timeoutFuture =
           channel.eventLoop().schedule(this::onTimeout, timeoutMillis, TimeUnit.MILLISECONDS);
     } else {
-      fail(describe() + ": error writing ", writeFuture.cause());
+      String message =
+          String.format("%s: failed to send request (%s)", describe(), writeFuture.cause());
```

Comment:
I'm adding the cause in the message string because these errors usually appear inside an `AllNodesFailedException`, and thus we don't usually see their cause printed in logs.